src/zcash-proving-service.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "streams.h"
+#include "util.h"
+#include "version.h"
+#include "zcash/JoinSplit.hpp"
+#include "zcash/Proof.hpp"
+
+#include <vector>
+#include <zmq.hpp>
```

Comment:
This should have been handled by the depends system. Were you using `./zcutil/build.sh`? I'll run a CI test to check it works regardless.